lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -28,19 +28,22 @@
 MAX_MISC_FEATURE_SIZE = 10000
 MAX_PARENT_LOOKUPS = 5
 
+# catalog params
+MAX_THREADS_DEFAULT = 10
+THREADS_PER_CPU_DEFAULT = 1
 
-class GenbankToGenome:
-    def __init__(self, config):
-        self.cfg = config
-        self.gi = GenomeInterface(config)
-        self.dfu = DataFileUtil(config.callbackURL)
-        self.aUtil = AssemblyUtil(config.callbackURL)
-        self.ws = Workspace(config.workspaceURL)
-        self._messages = []
```

Comment:
Looks like this was deleted. Is it not needed?

Reply:
You mean line 39? or 36-39?

Reply:
39

Reply:
is not used anywhere in this repo.

Reply:
:+1: